torch/csrc/distributed/rpc/rref_impl.cpp

```diff
@@ -80,6 +80,7 @@ UserRRef::UserRRef(
 
 UserRRef::~UserRRef() {
   try {
+    LOG(INFO) << "call deluser from destructor";
```

Comment:
Do we need this logging? With a large number of rrefs this would cause a lot of log spew. I'd suggest making this VLOG(1) if you want to keep it.